package.json

```diff
@@ -22,8 +22,8 @@
     "canvas-sequencer": "^3.0.6",
     "core-js": "^3",
     "express": "^4.17.1",
-    "socket.io": "^4.6.1",
-    "socket.io-client": "^4.6.1",
+    "socket.io": "^4.2.0",
+    "socket.io-client": "^4.2.0",
```

Comment:
I downgrade socket.io back to 4.2.0 so that we can get debug messages in the browser if we want. See https://github.com/socketio/socket.io-client/issues/1516